src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/offender/Aliases.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.offender
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import java.time.LocalDate
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Aliases(
```

Comment:
Maybe should be named `Alias` as it only holds one alias?

Reply:
Agree.